cob_hardware_config/cob4-7/config/image_flip_params_head_cam.yaml

```diff
@@ -1,6 +1,6 @@
 # rotation mode (0=fixed angle as defined  by rotation_angle, 1=automatic upright rotation against gravity using reference coordinate frame with upwards directed z-axis, 2=as 1 but image rotation only in 90deg steps -> faster)
 # int
-rotation_mode: 1
+rotation_mode: 0
```

Comment:
we should keep mode 1 here as we'd like to flip the image as soon as the torso starts bending.